stl/inc/xthreads.h

```diff
@@ -126,6 +125,7 @@ int __cdecl _Smtx_try_lock_exclusive(_Smtx_t*);
 int __cdecl _Smtx_try_lock_shared(_Smtx_t*);
 void __cdecl _Smtx_unlock_exclusive(_Smtx_t*);
 void __cdecl _Smtx_unlock_shared(_Smtx_t*);
+void __cdecl _Thrd_sleep_for(long long /*ns*/); // TRANSITION: defined in sharedmutex.cpp for convenience

```

Comment:
I still strongly disagree with an interface in `ns` given that the underlying bits ultimately accept `FILETIME` units, as that sells the interface as doing something it ultimately can't do, and pays range loss and scaling penalties to do so. In the hypothetical future a new Windows API exists that accepts nanos exists, by all means we can add one. But this status quo has been the status quo since NT's inception in 1993 and I don't see it changing for the foreseeable future.

(nanos made sense for Linux because they wanted to speak `struct timespec` which separates out a separate nanos field from `time_t`, with most software only needing to consider the `time_t`. NT never internally used `time_t`, and choosing 100ns increments gave a simple 64 bit counter enough range that they could set the epoch back at the start of the 400 year Gregorian calendar cycle at the time (January 1 1601) while still having ~20'000+ years of future range. Given this framing I don't see any reason NT would want to adopt nanos.)